src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure [] figures = new Figure[6];

```

Comment:
```suggestion
        Figure[] figures = new Figure[6];
```
if you want you can also extract 6 to ARRAY_LENGTH constant